docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss

```diff
@@ -24,8 +24,26 @@
   padding-top: hb-calculate-rems(40px);
 }
 
-// Add margin bottom to all paragraph rows
+.ptype-hs-text-area,
```

Comment:
@kaseybon after our discussion today, I want to leave it to you to decide whether we should leave this work in here, or just implement this later with CSS grid for both our grid system, and also the counted rows so we have some consistency.

I can implement what you decide. I decided not to add any other code because I had no helpful solutions I came up with!

Reply:
add 1/2 spacing unit of margin bottom to grid

Reply:
run backstop one more time